pfunk/__main__.py

```diff
@@ -297,7 +298,13 @@ def investigate(args):
         pfunk.tests.plot(name, args.show)
 
 
+class CleanFileAction(argparse.Action):
```

Comment:
A docstring would be good, I've never seen argparse.Action subclasses used before :-)